README.md

```diff
@@ -138,6 +138,15 @@ If you have any questions or discussion topics, start a [GitHub Discussion][8].
 ## Contributing
 Contributions are always welcome. Please read [Contributing.md](CONTRIBUTING.md) if you are considering making contributions.
 
+## Related projects
+- [i3-resurrect](https://github.com/JonnyHaystack/i3-resurrect) - "Simple solution to saving and restoring i3 workspaces"
```

Comment:
This project is already explained in the 'Similar Software' section (see [here](https://github.com/jdholtz/i3-restore/blob/develop/README.md#i3-resurrect)) so I think it can be removed